### DISCUSSION TITLE
[Feature Request] Add CLI option for base directory scaffolding

### DISCUSSION THREAD

**Discussion by norbux** [99designs/gqlgen#1887] [Ideas]
### What happened?
I would like to have a way to instruct `init` and `generate` CLIs to put the directories and files in a specific location different than the root directory. Consider this project structure:

```
cmd
- graphqlserver
pkg
- pkg-a
- pkg-b
go.mod
go.sum
```

Would be great to have some `--base-dir` option to specify, for example `cmd/graphqlserver` so that `init` and `generate` scaffold all the necessary files in there instead of the root directory.

I know there already are CLI flags for config, schema and server in init.go but these options do not solve the situation completely.

I already have this feature implemented in a fork and I would like to make a PR if you are interested.

Thanks in advance.

### Minimal graphql.schema and models to reproduce
N/A
### versions
 - `gqlgen version` 0.14.0
 - `go version` 1.17.5

**Comment by frederikhors** [comment]
@StevenACoffman what do you think to move issues like this in https://github.com/99designs/gqlgen/discussions/categories/ideas?

What does the whole team think of feature requests like this?
